src/proto/Binance.proto

```diff
@@ -181,20 +179,32 @@ message TimeRelockOrder {
     int64 lock_time = 5;
 }
 
+// Message for TimeUnlock order
 message TimeUnlockOrder {
     bytes from_address = 1; // owner address
     int64 id = 2; // order ID
 }
 
-// Input data necessary to create a signed order.
+// Input data necessary to create a signed transaction.
 message SigningInput {
+    // Chain ID
     string chain_id = 1;
+
+    // Source account number
     int64 account_number = 2;
+
+    // Sequence number (account specific)
     int64 sequence = 3;
+
     int64 source = 4;
```

Comment:
`source` field is missing: https://github.com/bnb-chain/BEPs/blob/master/BEP10.md